examples/modules.ts

```diff
@@ -510,7 +538,7 @@ export const moduleInstancesLookup = async (
 ): Promise<any[]> => {
   // Create a Security Token Instance
 
-  const securityToken = await polymathAPI.tokenFactory.getSecurityTokenInstanceFromTicker(ticker ? ticker : '');
+  const securityToken = await polymathAPI.tokenFactory.getSecurityTokenInstanceFromTicker(ticker || '');
```

Comment:
Just how it was coded. We use a browser prompt to get the ticker string so I default to '' so that it would notify the user gracefully as opposed to passing undefined around

Reply:
In that case it's better to just make `ticker` have a default assignment.

In both the `addModule` and `moduleInstancesLookup` functions, the signatures should look like this

```
async (
  polymathAPI: PolymathAPI,
  { ticker = '', moduleName }: GetAttachedModulesParams,
  /* ... other params */
): Promise<any[]>
```

And then you can remove that `ticker || ''` syntax from both of them

Reply:
Should I modify it in this branch?

Reply:
Yes